routes/transaction.go

```diff
@@ -2192,6 +2193,341 @@ func (fes *APIServer) SendDiamonds(ww http.ResponseWriter, req *http.Request) {
 	}
 }
 
+type DAOCoinOperationTypeString string
+
+const (
+	DAOCoinOperationStringMint                            DAOCoinOperationTypeString = "mint"
+	DAOCoinOperationStringBurn                            DAOCoinOperationTypeString = "burn"
+	DAOCoinOperationStringUpdateTransferRestrictionStatus DAOCoinOperationTypeString = "update_transfer_restriction_status"
+	DAOCoinOperationStringDisableMinting                  DAOCoinOperationTypeString = "disable_minting"
+)
+
+type TransferRestrictionStatusString string
+
+const (
+	TransferRestrictionStatusStringUnrestricted            TransferRestrictionStatusString = "unrestricted"
+	TransferRestrictionStatusStringProfileOwnerOnly        TransferRestrictionStatusString = "profile_owner_only"
+	TransferRestrictionStatusStringDAOMembersOnly          TransferRestrictionStatusString = "dao_members_only"
+	TransferRestrictionStatusStringPermanentlyUnrestricted TransferRestrictionStatusString = "permanently_unrestricted"
+)
+
+// DAOCoinRequest ...
+type DAOCoinRequest struct {
+	// The public key of the user who is performing the DAOCoin Txn
+	UpdaterPublicKeyBase58Check string `safeForLogging:"true"`
+
+	// The public key or username of the profile whose DAO coin the transactor is trying to transact with.
+	ProfilePublicKeyBase58CheckOrUsername string `safeForLogging:"true"`
+
+	// Whether this is a "mint", "burn" or "disable_minting" transaction
+	OperationType DAOCoinOperationTypeString `safeForLogging:"true"`
+
+	// Coins
+	CoinsToMintNanos uint256.Int `safeForLogging:"true"`
```

Comment:
Does... this work? @lazynina

Reply:
Like can json serialize uint256?

Reply:
Yeah it becomes a hex string!!

Reply:
Hmmm looks like... maybe it works if you encode the things as a uint256? Not sure how to solve this problem...

https://github.com/holiman/uint256/blob/46b8ad94199afbcc12f2ca653481891165d829d5/conversion.go#L470